rllib/env/policy_client.py

```diff
@@ -246,7 +252,7 @@ def run(self):
                 samples = self.rollout_worker.sample()
                 metrics = self.rollout_worker.get_metrics()
                 logger.info("Sending batch of {} steps back to server.".format(
-                    samples.count))
+                    samples.total()))
```

Comment:
Consider reporting both (env steps=X, agent steps=Y), if they are different.

Reply:
done.

Reply:
The thing is: I don't think `count` is always kept up to date inside `MultiAgentBatch`.
`total()` on the other hand sums up all sub SampleBatches's counts, which actually are kept up to date (because in single SampleBatch, an agent step cannot be "skipped", like in the case of MultiAgent). count is probably not a good metric here at all. Not sure, whether we require or use this anywhere else, though.